core/src/components/datetime/utils/warn.ts

```diff
@@ -0,0 +1,52 @@
+import { printIonWarning } from '@utils/logging';
```

Comment:
~ Optional: I think `validate.ts` would be a better name for what this file is responsible for and the functions within it. 

"Warn" is what is happening as a side-effect of validating the data coming in.

Reply:
updated in 85a7d73763